packages/website/src/components/SplitPaneResize.css

```diff
@@ -0,0 +1,49 @@
+.Resizer {
+  background: #000;
+  opacity: 0.2;
+  z-index: 1;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  -moz-background-clip: padding;
+  -webkit-background-clip: padding;
+  background-clip: padding-box;
```

Comment:
I spy copypasta 🍝 😄 https://github.com/tomkp/react-split-pane#example-styling

These styles are fine to start with, but a few changes to make in addition to using our CSS variables:

* We don't need `-moz-*` or `-webkit-` prefixes; they're supported by stable browser versions now and we have autoprefixing set up anyway I'm pretty sure
* We don't have any horizontal resizers, so that CSS can be removed altogether
* We use CSS modules elsewhere. `SplitPane` components can take in `className` & `resizerClassName` props, so let's use those & switch this file to CSS modules.

Reply:
Changed.